Bio/PDB/Structure.py

```diff
@@ -7,9 +7,40 @@
 
 """The structure class, representing a macromolecular structure."""
 
+from typing import NamedTuple
+
+from Bio.PDB.Atom import Atom
+from Bio.PDB.Chain import Chain
 from Bio.PDB.Entity import Entity
 
 
+class DisulfideBond(NamedTuple):
```

Comment:
This is a funny way of creating a namedtuple. I would rather avoid importing `typing` only to use the `NamedTuple` as a subclass. If you prefer this syntax, why not use a frozen dataclass? Otherwise, I believe the "standard" way of declaring namedtuple is simpler.